src/index.ts

```diff
@@ -54,6 +54,16 @@ commander
     `selects the service to be used for translation`,
     'google-translate',
   )
+  .option(
+    '-g, --glossaries <glossariesDir>',
+    `set the glossaries folder to be used by DeepL`,
+    './i18n/glossaries',
```

Comment:
By default, this scripts looks for locale files in the current working directory (see line 30), so the default value of this option might clash with that. It might make sense to not supply a default value here to avoid that.

Reply:
Yes, we added this while developing because we couldn't figure out how to actually call the script and pass values to it. Simply providing a default, however, worked. When developing, how do I call the script? When running `npm run start` and passing the options there, they are not picked up.

Reply:
Thanks, @leolabs, this worked! The code now successfully uses glossaries to translate with DeepL. Could you do another review when you get the chance? I can't request another review. Probably because it wasn't me who created the initial PR.